project/gen2.scala

```diff
@@ -32,13 +32,17 @@ object gen2 {
   implicit val AngleMeta: Meta[Angle] =
     Meta[Double].xmap(Angle.fromArcsecs, _.toArcsecs)
 
+  implicit val ZoneIdMeta: Meta[ZoneId] =
+    Meta[String].xmap(ZoneId.of, _.toString)
```

Comment:
This is only used at build time so we will know when we try to compile, but it's also possible that the runtime JVM won't have the timezones, in which case gem will just crash. It can't run without this information.